dintero-checkout-for-woocommerce.php

```diff
@@ -234,12 +234,21 @@ function () {
 		 * @return void
 		 */
 		public function wc_zero_decimal_notice() {
+
+			if ( get_user_meta( get_current_user_id(), 'dismissed_wc_zero_decimal_notice', true ) ) {
```

Comment:
We should probably add dintero to this id, so we don't accidentally conflict with other plugins here. As well as row 249.